### PR TITLE
styling, readme, and map picking improvements

### PR DESCRIPTION
**Pull request by pricealexandra**

### CODE REVIEW

File: components/map/map.tsx [hunk 1]
```diff
@@ -22,26 +22,26 @@ const DeckGLOverlay = props => {
...
-  }
-};
-
 type MapProps = {
   bikeShareData: BikeShareFeature[];
   amtrakData: AmtrakResponse;
 };
 
 const Map: React.FC<MapProps> = ({ bikeShareData, amtrakData }) => {
+  const getTooltip = useCallback(
```
Comment (pricealexandra) [inline components/map/map.tsx:31]: I know this wasn't on the list but it seems like enough of a common use case that it was worth adding. also didn't know what else to do at the end of the day.

Comment (paigewilliams) [reply components/map/map.tsx:31]: looks great!

File: components/panel/panel.module.css [hunk 2]
```diff
@@ -2,10 +2,16 @@
...
   padding: 0 1rem;
-  overflow-y: scroll;
-  h2 {
+  h3 {
     margin: 0;
-    font-size: 1.5rem;
+    padding: 1rem 0 0 0;
+    font-size: 1.25rem;
+  }
+  p{
```
Comment (paigewilliams) [inline components/panel/panel.module.css]: a super nit: I think we need a space between `p` and `{`

**Review by paigewilliams — APPROVED** [review]
✨

File: components/panel/panel.module.css (hunk 2 above)
Comment (pricealexandra) [reply components/panel/panel.module.css]: thank you for noticing, fixed!

**Pull request merged**